packaging/docker/run.sh

```diff
@@ -13,11 +13,79 @@ if [ ! "${DO_NOT_TRACK:-0}" -eq 0 ] || [ -n "$DO_NOT_TRACK" ]; then
   touch /etc/netdata/.opt-out-from-anonymous-statistics
 fi
 
+create_group=
+remove_group=
+create_user=
+remove_user=
+user_in_group=
+
+if [ -n "${DOCKER_USR}" ]; then
+  NETDATA_USER="${DOCKER_USR}"
+fi
+
+if getent group netdata > /dev/null; then
+  existing_gid="$(getent group netdata | cut -d ':' -f 3)"
+
+  if [ "${existing_gid}" != "${NETDATA_GID}" ]; then
+    echo "Netdata group ID mismatch (expected ${NETDATA_GID} but found ${existing_gid}), the existing group will be replaced."
+    remove_group=1
+    create_group=1
+  fi
+else
+  echo "Netdata group not found, preparing to create one with GID=${NETDATA_GID}."
+  create_group=1
+fi
+
+if [ -n "${remove_group}" ]; then
+  delgroup netdata netdata
```

Comment:
> Usage: delgroup [USER] GROUP

Is `netdata` user is the only option at this point?

Reply:
No, but the group needs to be empty for `delgroup` to let us delete it. The only user that _should_ exist in the group should be `netdata`.

We actually do need an additional check here though I guess, to make sure that the group and password databases are writable before we try to do this.